lib/manager/sbt/extract.ts

```diff
@@ -20,6 +20,11 @@ const isScalaVersion = (str: string): boolean =>
 const getScalaVersion = (str: string): string =>
   str.replace(/^\s*scalaVersion\s*:=\s*"/, '').replace(/"\s*$/, '');
 
+const normalizeScalaVersion = (str: string): string =>
```

Comment:
Can the reasoning for this be documented with a code comment and/or with a reference to documentation somewhere?

Reply:
Done